File: src/main/java/pl/yalgrin/playnite/simplesync/web/HealthResource.java
package pl.yalgrin.playnite.simplesync.web;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;
import reactor.core.publisher.Mono;

@RestController
@RequestMapping("/api/health")
public class HealthResource {
    @GetMapping
    public Mono<String> testConnection() {
        return Mono.just("OK");
    }
}

File: src/main/kotlin/pl/yalgrin/playnite/simplesync/web/health/HealthResource.kt
package pl.yalgrin.playnite.simplesync.web.health

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController
import reactor.core.publisher.Mono

@RestController
@RequestMapping("/api/health")
class HealthResource {
    @GetMapping
    fun testConnection(): Mono<String> {
        return Mono.just("OK")
    }
}
